addons/sale/models/utm_campaign.py

```diff
@@ -7,8 +7,10 @@ class UtmCampaign(models.Model):
     _inherit = 'utm.campaign'
     _description = 'UTM Campaign'
 
-    quotation_count = fields.Integer('Quotation Count', groups='sales_team.group_sale_salesman', compute="_compute_quotation_count")
-    invoiced_amount = fields.Integer(default=0, compute="_compute_sale_invoiced_amount", string="Revenues generated by the campaign")
+    quotation_count = fields.Integer('Quotation Count', groups='sales_team.group_sale_salesman',
+        compute="_compute_quotation_count", compute_sudo=True)
+    invoiced_amount = fields.Integer(default=0, compute="_compute_sale_invoiced_amount", compute_sudo=True,
+        string="Revenues generated by the campaign", groups='sales_team.group_sale_salesman')
```

Comment:
```suggestion
    invoiced_amount = fields.Integer("Revenues generated by the campaign", groups='sales_team.group_sale_salesman',
        compute='_compute_sale_invoiced_amount', compute_sudo=True)
```
Do you mind applying the same order of field params for both fields and remove the useless default while you're modifying those fields? :)

Reply:
+1 (would say: string, compute, compute_sudo, groups)